src/acl/external/time_quota/ext_time_quota_acl.8

```diff
@@ -35,14 +35,8 @@ Pauses shorter than this value will be counted against the quota, longer ones ig
 Default is 300 seconds (5 minutes).
 .
 .if !'po4a'hide' .TP
-.if !'po4a'hide' .B "\-l logfile"
-.B Filename
-where all logging and debugging information will be written. If none is given,
-then stderr will be used and the logging will go to Squids main cache.log.
```

Comment:
I still think we should try to keep existing `-l` functionality, extending Debug::NameThisHelper() if necessary, but I do not insist on that, especially if it is really difficult to preserve that feature.

Reply:
Restored